tests/basics/task.yaml

```diff
@@ -1,7 +1,8 @@
 summary: Check that basic interactions work
+restore: |
+    snap remove hello-world
```

Comment:
We need a mechanism that allows us to set per-task prepare/restore too. I'll add that today.

Meanwhile, please copy the suite "restore" script here, and restore the original logic below.